src/Resources/Users/index.ts

```diff
@@ -49,6 +49,12 @@ export default class Users extends AbstractResource {
         this.defaultAccess = 'member';
     }
 
+    public userWithStatuses: (...args: HttpClientConfig) => HttpClientResponse = this.createMethodFromPartialSpec({
+        access: 'auth',
+        method: 'GET',
+        path: '/user?balance=1&changeSubscription=1&cancelSubscription=1',
```

Comment:
Even though this is the reason we are adding this endpoint. This should still be added to the uwa. As there could be times we need the balance and not cancel subscription for example